File: homework4_1/src/main/java/com/example/homework4_1/AlertEmptyDialog.java
package com.example.homework4_1;

import android.app.Dialog;
import android.content.DialogInterface;
import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatDialogFragment;

public class AlertEmptyDialog extends AppCompatDialogFragment {

    @NonNull
    @Override
    public Dialog onCreateDialog(@Nullable Bundle savedInstanceState) {
        android.app.AlertDialog.Builder builder = new android.app.AlertDialog.Builder(getActivity());
        builder.setTitle("Attemption")
                .setMessage("Name can not be empty")
                .setPositiveButton("Ok",  null);
        return builder.create();
    }
}

File: homework5_2/src/main/java/com/example/homework5_2/AlertDialogs/AlertEmptyDialog.kt
package com.example.homework5_2.AlertDialogs

import android.app.AlertDialog
import android.app.Dialog
import android.os.Bundle
import androidx.appcompat.app.AppCompatDialogFragment

class AlertEmptyDialog : AppCompatDialogFragment() {
    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog =
        AlertDialog.Builder(activity)
            .setTitle("Attemption")
            .setMessage("Name cant be empty")
            .setPositiveButton("Ok", null)
            .create()

}